mantidproject/mantid
# Improve BSpline and CubicSpline Documentation

The documentation for BSpline and CubicSpline fitting functions is lacking explanations of what these functions are and how they can be used in the sense of fitting.

**To test**
- Requires you to build qt-help pages
- Review Documentation of BSpline and CubicSpline
- Make sure Doc Tests pass

[Release Notes](http://www.mantidproject.org/ReleaseNotes_3_7_UI_Changes#Documentation)

Fixes #15015
